server/views/pages/plan-summary.njk

```diff
@@ -132,7 +132,6 @@
             {% elseif data.plan.agreementStatus === 'DO_NOT_AGREE' %}
                 <p class="govuk-body">{{ locale.notAgreePlan }}</p>
             {% elseif data.plan.agreementStatus === 'DRAFT' %}
```

Comment:
I think you can remove this elseif line now that there's no output with this agreementStatus